requirements.txt

```diff
@@ -7,6 +7,6 @@ h5py==2.10.0
 opencv-contrib-python==4.4.0.42
 opencv-python==4.4.0.42
 matplotlib
-numpy
+numpy=1.17.0
```

Comment:
permit anything newer than 1.17
```suggestion
numpy>=1.17.0
```

Reply:
I tested it with both 1.17.0 and 1.20.0 and the numpy version has to be less than 1.20.0. I would suggest using 1.17.0 here.

Reply:
we may need to update the training script then. is it obvious if the numpy version is wrong? Like is it a crash, or does it fail silently and produce inaccurate results?

Reply:
Its a crash, doesn't run 😿

Reply:
Ok, that's good then. I was worried if we run in Colab (because they have pre-installed versions of all the software), it might silently fail. We can add more parameters to our training script to install a specific version of `numpy`if we encounter any issues.